core/api/src/main/java/com/alipay/sofa/rpc/protocol/telnet/HelpTelnetHandler.java

```diff
@@ -60,7 +59,14 @@ public String telnet(AbstractChannel channel, String message) {
 
     @Override
     public String getDescription() {
-        return "show all support commands!" + LINE + "Usage:\thelp" + LINE + "\thelp [cmd]";
+        StringBuilder description = new StringBuilder();
+        description.append("show all support commands!" + LINE + "Usage:\thelp" + LINE + "\thelp [cmd]");
```

Comment:
Consider how to how to extend the description. Once we add a telnet handler, this method need to be changed. It breaks the **Open Closed Principle**.